fud/fud/json_to_dat.py

```diff
@@ -1,21 +1,197 @@
 import json
 import numpy as np
 from pathlib import Path
-
+from .errors import ConversionError
 
 # Converts `val` into a bitstring that is `bw` characters wide
 def bitstring(val, bw):
     # first truncate val by `bw` bits
     val &= (2**bw - 1)
     return '{:x}'.format(val)
 
+# Converts `val` in binary representation to a bitstring that is `bw` characters wide
+def bitstring2(val, bw):
```

Comment:
how is this function different from `bitstring`? is it possible to unify `bistring` and `bitstring2` somehow

Reply:
bitstring2 takes in the string representation of the value and turns it into bitstring  since my decimal to fixed point function would do something like 11.125 -> '01011001'. I think I can just use the original bitstring and do the conversion from '01011001' to number before calling bitstring.